app/uk/gov/hmrc/gform/sharedmodel/formtemplate/InternalLink.scala

```diff
@@ -29,6 +29,7 @@ object InternalLink {
   case object NewSession extends InternalLink
   case object SignOut extends InternalLink
   case class PageLink(id: PageId) extends InternalLink
+  case class DownloadFile(fileName: String, fileExtension: FileExtension) extends InternalLink
```

Comment:
It would be nice to raise an error if nonexistent file is specified here, can we do that as part of FormTemplate validation?